### ISSUE TITLE
[Bug]: using an unstable build of TA breaks version checking

### ISSUE THREAD

**Issue by Styloy** [tubearchivist/tubearchivist-plex#54]
### I've read the documentation.

- [X] I'm running the latest version of the TubeArchivist Plex Integration Scanner and Agent. I have read the latest release notes and installation instructions.

### Operating System

TrueNAS (should be platform agnostic)

### Your Bug Report

there is a parsing issue in test_ta_connection (in both scanner and agent), when using an unstable build of TA it chokes due to "-unstable" being appended in the response of /api/ping. 

on my instance, I've changed the following as a quick and dirty fix. I can PR this if you feel this is suitable.
ta_version = [int(x) for x in response['version'][1:].split(".")]
to
ta_version = [int(x) for x in response['version'][1:].split(".").rstrip("-unstable")]

### Relevant log output

```Shell
2024-03-22 11:36:10,495 [TubeArchivist Scanner] INFO - File matches expected filename layout.
2024-03-22 11:36:10,496 [TubeArchivist Scanner] ERROR - TubeArchivist instance version is unknown or unset. Please review the logs further and ensure that there is connecti>
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - Processing file with scanner: RpepxkWU814.mp4
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - File matches expected filename layout.
2024-03-22 11:36:10,497 [TubeArchivist Scanner] ERROR - TubeArchivist instance version is unknown or unset. Please review the logs further and ensure that there is connecti>
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - Processing file with scanner: RvdelRCnInw.mp4
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - File matches expected filename layout.
2024-03-22 11:36:10,497 [TubeArchivist Scanner] ERROR - TubeArchivist instance version is unknown or unset. Please review the logs further and ensure that there is connecti>
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - Processing file with scanner: Rwi8nYhU9qU.mp4
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - File matches expected filename layout.
2024-03-22 11:36:10,497 [TubeArchivist Scanner] ERROR - TubeArchivist instance version is unknown or unset. Please review the logs further and ensure that there is connecti>
2024-03-22 11:36:10,497 [TubeArchivist Scanner] INFO - Processing file with scanner: RyE53hYsSig.mp4
```

**Comment by lamusmaser** [comment]
Noted. I did not test with Unstable, so I'll add that logic into the next release.

**Comment by lamusmaser** [comment]
Incorporated fix into v0.1.5. This should no longer be an issue once that version is released.

**Comment by lamusmaser** [comment]
v0.1.5 is now released and available. Closing until someone reports this as a continued issue.
